doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,5 +1,17 @@
 # Upgrade Notes
 
+## 6.1.2
```

Comment:
BC breaking change?

Reply:
Don't think so ... nothing changes if you're using a custom session config. 
If you're using the default, it will continue to work by using PHP's default config.

Reply:
I just thought it is worth to mention in the upgrade notes.